db-scripts/src/main/resources/migration.sql

```diff
@@ -444,4 +444,11 @@ WHERE c2.`CANCER_STUDY_ID` = c1.`CANCER_STUDY_ID` AND c2.`SAMPLE_ID` = c1.`SAMPL
 ABS(c2.`SEGMENT_MEAN`) >= 0.2) / SUM(`END`-`START`)) AS `VALUE` FROM `copy_number_seg` AS c1, cancer_study WHERE 
 c1.`CANCER_STUDY_ID` = cancer_study.`CANCER_STUDY_ID` GROUP BY cancer_study.`CANCER_STUDY_ID`, `SAMPLE_ID` HAVING SUM(`END`-`START`) > 0;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.5.0";
\ No newline at end of file
+UPDATE info SET DB_SCHEMA_VERSION="2.5.0";
+
+##version 2.6.0
+-- modify fkc for gistic_to_gene
+ALTER TABLE gistic_to_gene DROP FOREIGN KEY gistic_to_gene_ibfk_2;
```

Comment:
Do we know that the name for the foreign key *gistic_to_gene_ibfk_2* will be found on all database installations? I think that foreign key constraint operations are silently ignored on MyISAM engine databases ... and perhaps this foreign key naming pattern is canonical across all INNODB engine installations. But I would double check this before we commit to this schema update (since schema version changes are immutable once committed to).
If we are wrong about this, we may need to create documentation for installations on systems where the foreign key name differs from this pattern ... helping them to manually adjust the foreign key without the migration.sql script.

Maybe (in a different PR) we would want to assign chosen names to all foreign key constraints to allow future deletions / modifications. But maybe this is not possible if the FKC name is set implicitly.

Reply:
@sheridancbio yes, the name of the constraint is `gistic_to_gene_ibfk_2` - the constraint wasn't actually missing but it was missing the `DELETE ON CASCADE`

Reply:
I hope no one is still on myisam...?